robottelo/test.py

```diff
@@ -132,13 +134,26 @@ class UITestCase(TestCase):
     def setUpClass(cls):  # noqa
         """Make sure that we only read configuration values once."""
         super(UITestCase, cls).setUpClass()
-        cls.katello_user = settings.server.admin_username
-        cls.katello_passwd = settings.server.admin_password
+
+        cls.katello_default_user = settings.server.admin_username
```

Comment:
default_password is bein used in test_login, So i thought that keeping user woulf be useful to keep the pattern maybe for future cases